StorageAppUITests/StorageAppUITests.m

```diff
@@ -0,0 +1,201 @@
+//
+//  StorageAppUITests.m
+//  StorageAppUITests
+//
+//  Created by Ashwin Kumar on 10/12/21.
+//  Copyright © 2021 CA Technologies. All rights reserved.
+//
+
+#import <XCTest/XCTest.h>
+
+#define INTERRUPT_MONITOR_DESCRIPTION @"TestInitialScreenHandler"
+#define PERMISSIONS_ALLOW_BUTTON @"Allow While Using App"
+#define LOCAL_STORAGE_UPDATE_SUCCESS_TEXT @"LocalStorageItem updated Successfully"
+#define LABEL_OK @"OK"
+#define LABEL_FAILURE @"Failure"
+#define LABEL_LOCAL_STORAGE @"Local Storage"
+#define LABEL_APP_USER_SEGMENT @"AppUser Seg."
+#define LABEL_APP_SEGMENT @"App Segment"
+#define LABEL_SHARE @"Share"
+#define LABEL_ADD_STRING_OBJECT @"Add data of type String"
+#define LABEL_ADD_IMAGE_OBJECT @"Add data of type Image"
+#define LABEL_UPDATE @"Update"
+#define LABEL_DELETE @"Delete"
+#define LABEL_UPDATED_OBJECT @"Testing Update"
+#define LABEL_NEW_OBJECT_FORMAT_STRING @"NewLocalStorage%lu"
+
+@interface StorageAppUITests : XCTestCase
+
+@property(nonatomic,strong) XCUIApplication *app;
+
+@end
+
+@implementation StorageAppUITests
+
+- (void)setUp {
+    // Put setup code here. This method is called before the invocation of each test method in the class.
+
+    // In UI tests it is usually best to stop immediately when a failure occurs.
+    self.continueAfterFailure = NO;
+    _app = [self initializeApp];
+
+    // In UI tests it’s important to set the initial state - such as interface orientation - required for your tests before they run. The setUp method is a good place to do this.
+}
+
+- (void)tearDown {
+    // Put teardown code here. This method is called after the invocation of each test method in the class.
+}
+
+- (XCUIApplication*) initializeApp {
+    XCUIApplication *app = [[XCUIApplication alloc] init];
+    [self addUIInterruptionMonitorWithDescription:INTERRUPT_MONITOR_DESCRIPTION handler:^BOOL(XCUIElement * _Nonnull interruptingElement) {
+        
+        // handling permissions popup
```

Comment:
Please follow our commenting style to be uniform across, for example :
//
// Invoke protected endpoint
//